website/src/content/api-reference/commercetools-frontend-application-shell.mdx

```diff
@@ -213,6 +213,8 @@ const useExternalApiFetcher = () => {
       },
       // Optionally define the audience policy
       audiencePolicy: 'forward-url-full-path',
+      // Optionally ask for users permissions to be included in exchange JWT
```

Comment:
```suggestion
      // Include user permissions in the JWT (optional)
```

Reply:
@CarlosCortizasCT, this applies to all such comments.

Reply:
I partially updated it in this commit: 9d0903a0fd069c9b485bb1bde48621e7d29fddd1

However, I believe the new sentence suggests the JWT is part of what the `createApolloContextForProxyForwardTo` function creates and that's not the case. This new property will make the result of this function call to contain the `X-Forward-To-Claims` header with the `permissions` value so the MC Gateway will include user's permission in the exchange JWT that will get sent along the request to the external service. 
That's why I think we might want to keep the reference to the exchange token. Perhaps we need to find a better way to explain it.

Reply:
Updated in this commit: c8053312999955e37e23bc1c73a9fb8896af5314